src/transformers/dynamic_module_utils.py

```diff
@@ -532,19 +532,27 @@ def resolve_trust_remote_code(trust_remote_code, model_name, has_local_code, has
         if has_local_code:
             trust_remote_code = False
         elif has_remote_code and TIME_OUT_REMOTE_CODE > 0:
-            signal.signal(signal.SIGALRM, _raise_timeout_error)
-            signal.alarm(TIME_OUT_REMOTE_CODE)
-            while trust_remote_code is None:
-                answer = input(
-                    f"Loading {model_name} requires to execute some code in that repo, you can inspect the content of "
-                    f"the repository at https://hf.co/{model_name}. You can dismiss this prompt by passing "
-                    "`trust_remote_code=True`.\nDo you accept? [y/N] "
+            try:
+                signal.signal(signal.SIGALRM, _raise_timeout_error)
+                signal.alarm(TIME_OUT_REMOTE_CODE)
+                while trust_remote_code is None:
+                    answer = input(
+                        f"Loading {model_name} requires to execute some code in that repo, you can inspect the content of "
+                        f"the repository at https://hf.co/{model_name}. You can dismiss this prompt by passing "
+                        "`trust_remote_code=True`.\nDo you accept? [y/N] "
+                    )
+                    if answer.lower() in ["yes", "y", "1"]:
+                        trust_remote_code = True
+                    elif answer.lower() in ["no", "n", "0", ""]:
+                        trust_remote_code = False
+                signal.alarm(0)
+            except AttributeError:
+                # OS which does not support signal.SIGALRM
+                raise ValueError(
+                    "Loading this model requires you to execute execute some code in that repo in that repo on your "
```

Comment:
There is `in that repo` duplicated in this line.

BTW, could we also use f-string to have `model_name`, and also the repo. link. It would be even better.